### PR TITLE
Add priority fee

### PR DESCRIPTION
**Pull request by evgeny-stakewise**

### CODE REVIEW

File: src/common/execution.py [hunk 1]
```diff
@@ -149,8 +149,46 @@ async def get_oracles() -> Oracles:
     )
 
 
-async def check_gas_price() -> bool:
-    max_fee_per_gas = await _get_max_fee_per_gas()
+async def get_tx_params() -> TxParams:
```
Comment (cyc60) [inline src/common/execution.py:152]: this tx params relevant only to validator's registration transactions. MB move to validators/execution.py and rename to smth like get_validator_tx_params?

Comment (cyc60) [reply src/common/execution.py:152]: Or this actual to high priority transaction? Also can be expressed in the func name

File: src/common/execution.py [hunk 2]
```diff
@@ -149,8 +149,46 @@ async def get_oracles() -> Oracles:
...
+
+async def _calc_priority_fee() -> Wei:
+    """
+    reference: "high" priority value from https://etherscan.io/gastracker
+    """
+    num_blocks = 10
+    percentile = 80.0
+    history = await execution_client.eth.fee_history(num_blocks, 'pending', [percentile])
+    validator_rewards = [r[0] for r in history['reward']]
+    logger.info(
```
Comment (cyc60) [inline src/common/execution.py:176]:
```suggestion
    logger.debug(
```

File: src/common/execution.py [hunk 3]
```diff
@@ -149,8 +149,46 @@ async def get_oracles() -> Oracles:
...
+    logger.debug('tx_params %s', tx_params)
+
+    return tx_params
+
+
+async def _calc_priority_fee() -> Wei:
+    """
+    reference: "high" priority value from https://etherscan.io/gastracker
+    """
+    num_blocks = 10
```
Comment (cyc60) [inline src/common/execution.py:172]: might this require some fine tuning? mb move to settings?

File: src/common/execution.py (hunk 1 above)
Comment (evgeny-stakewise) [reply src/common/execution.py:152]: get_tx_params is used by check_gas_price located in common/execution

If I move get_tx_params to validators dir then common/execution file will import validators/execution. The imports were done in another order so far.

Comment (evgeny-stakewise) [reply src/common/execution.py:152]: get_tx_params name is generic, so yes, high priority should be expressed

**Review by cyc60 — APPROVED** [review]

**Pull request merged**